Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs

```diff
@@ -16,6 +16,7 @@
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.EventSystems;
+using UnityEngine.Windows.Speech;
```

Comment:
Please strap this in a Windows #if, 
This should Not be in the core input manager. Will need to refactor this out to another layer later. 

We ideally should never have platform specific dependencies in a manager

Reply:
Just wrap it for now, good enough for summit. Won't be enough time now to make it fully cross platform.